src/graphql/thegraph/PoolData.ts

```diff
@@ -0,0 +1,64 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+import { ChainId } from '@uniswap/sdk-core'
+import gql from 'graphql-tag'
+import { useMemo } from 'react'
+
+import { usePoolDataQuery } from './__generated__/types-and-hooks'
+import { CHAIN_SUBGRAPH_URL } from './apollo'
+
+gql`
+  query PoolData($poolId: [ID!]) {
+    data: pools(where: { id_in: $poolId }, orderBy: totalValueLockedUSD, orderDirection: desc, subgraphError: allow) {
+      id
+      feeTier
+      liquidity
+      sqrtPrice
+      tick
+      token0 {
+        id
+        symbol
+        name
+        decimals
+        derivedETH
+      }
+      token1 {
+        id
+        symbol
+        name
+        decimals
+        derivedETH
+      }
+      token0Price
+      token1Price
+      volumeUSD
+      volumeToken0
+      volumeToken1
+      txCount
+      totalValueLockedToken0
+      totalValueLockedToken1
+      totalValueLockedUSD
+    }
+    bundles(where: { id: "1" }) {
+      ethPriceUSD
+    }
+  }
+`
```

Comment:
copy/pasta from info. ?

Reply:
Basically, I just haven't integrated using blocks for different time windows yet https://github.com/Uniswap/v3-info/blob/master/src/data/pools/poolData.ts#L18